### PR TITLE
disable SSL Inspection

### LINKED ISSUE #11: Allow to turn ssl_verify off?
Otherwise this would often fail behind corporate proxies like so:

``` console
urllib.error.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:600)>
```

### PR DESCRIPTION
**Pull request by jeeftor**
Closes #11.

- Added an environment variable that allows you to disable SSL inspection - if for some reason you wanted to do that.

### CODE REVIEW

**Comment by jeeftor** [comment]
This pull request adds two features:
* Support for spaces in commands:  `tldr git pull` will work the same as `tldr git-pull` #38 
* ability to disable SLL inspection if you want to 

The CI isn't working correctly but i don't know if that matters.

**Comment by waldyrious** [comment]
@jeeftor the build is failing in Python 3.3 only -- perhaps you need to include some compatibility code?

```
tldr.py:25: in <module>

    ignore_ctx = ssl.create_default_context()

E   AttributeError: 'module' object has no attribute 'create_default_context'
```

**Comment by waldyrious** [comment]
Also, I'd prefer if you submitted the hyphen-joining code in a separate PR, if that wouldn't be too much of a hassle :)

**Comment by felixonmars** [comment]
I'm ok with the idea, however, it could use a better wording than `IGNORE_SSL`. How about `ALLOW_INSECURE`?

**Comment by zdroid** [comment]
@waldyrious Do we proceed with merging this one?

**Comment by MasterOdin** [comment]
@zdroid @waldyrious @felixonmars I updated the PR for the requested changes and fixed merge conflicts and I think this should be good to go?

**Comment by zdroid** [comment]
Good work here!
